platform/posix/transport/src/mbedtls_pkcs11_posix.c

```diff
@@ -176,29 +178,28 @@ static int32_t generateRandomBytes( void * pCtx,
  *
  * @param[in] pContext Caller TLS context.
  * @param[in] pLabelName PKCS #11 certificate object label.
- * @param[in] class PKCS #11 certificate object class.
  * @param[out] pCertificateContext Certificate context.
  *
- * @return Zero on success.
+ * @return True on success.
  */
-static int32_t readCertificateIntoContext( MbedtlsPkcs11Context_t * pContext,
-                                           char * pLabelName,
-                                           CK_OBJECT_CLASS class,
-                                           mbedtls_x509_crt * pCertificateContext );
+static bool readCertificateIntoContext( MbedtlsPkcs11Context_t * pContext,
+                                        char * pLabelName,
+                                        mbedtls_x509_crt * pCertificateContext );
 
 /**
- * @brief Helper for setting up potentially hardware-based cryptographic context.
+ * @brief Helper for configuring MbedTLS to use client key from PKCS #11.
  *
- * @param pCtx Caller context.
+ * @param pContext Caller context.
  * @param pPrivateKeyLabel PKCS #11 label for the private key.
  *
- * @return Zero on success.
+ * @return True on success.
  */
-static CK_RV initializeClientKeys( MbedtlsPkcs11Context_t * pCtx,
-                                   char * pPrivateKeyLabel );
+static bool initializeClientKeys( MbedtlsPkcs11Context_t * pContext,
+                                  char * pPrivateKeyLabel );
```

Comment:
Shouldn't constness be added to the label parameter?
```suggestion
                                 const char * pPrivateKeyLabel );
```